src/Collection/Collection.types.ts

```diff
@@ -48,3 +48,13 @@ export enum CollectionTypeFilter {
   STANDARD = 'standard',
   THIRD_PARTY = 'third_party'
 }
+
+export enum CollectionSort {
+  MOST_RELEVANT = 'MOST_RELEVANT',
+  NEWEST = 'NEWEST',
```

Comment:
I know this was already named like this, but now that we'll support both newest and oldest, just to keep it consistent with the rest of the enum options, what do you think if we name these ones as `CREATED_AT_DESC` and `CREATED_AT_ASC`? Then the front end will be the one translating newest and latest to `CREATED_AT_DESC` and ``CREATED_AT_ASC`